middleware/protect.js

```diff
@@ -32,7 +32,14 @@ function forceLogin (keycloak, request, response) {
 
   let uuid = UUID();
   let loginURL = keycloak.loginUrl(uuid, redirectUrl);
-  response.redirect(loginURL);
+  try {
+    // express.js not happy with next param
+    response.redirect(301, loginURL);
```

Comment:
I think we should use `302`. Looking into the [docs](http://expressjs.com/en/5x/api.html#res.redirect) and I think this is what we used to return before as it's the default value. Also IMHO `302` (found a.k.a moved temporarily) is more appropriate than `301` (moved permanently).

Reply:
sure, it does make sense, I will push an update for both of them

Reply:
Having second thoughts on this, If it's permanent redirect , it might skip browser checking the destination url that we modify on a next call 🤔 what do you think?

Reply:
Well of course, that's why we want to use `302`. 😉